build/tasks/validate.js

```diff
@@ -307,5 +301,170 @@ function validateRule({ tags, metadata }) {
   if (help.toLowerCase().includes(prohibitedWord)) {
     issues.push(`metadata.help can not contain the word '${prohibitedWord}'.`);
   }
+
+  issues.push(...findTagIssues(tags));
   return issues;
 }
+
+const uniqueTags = ['ACT', 'experimental', 'review-item', 'deprecated'];
+
+const categories = [
+  'aria',
+  'color',
+  'forms',
+  'keyboard',
+  'language',
+  'name-role-value',
+  'parsing',
+  'semantics',
+  'sensory-and-visual-cues',
+  'structure',
+  'tables',
+  'text-alternatives',
+  'time-and-media'
+];
+
+const standardsTags = [
+  {
+    name: 'WCAG',
```

Comment:
Seems odd to leave out validating wcag tags since a rule can only belong to one wcag level. E.g. you can't have `wcag2aa` and `wcag2a` on the same rule. Also I think wcag tags should come before all other standard tags now that we're already enforcing an order.

Reply:
Oh you're right. I was basing it solely on the `if` statements looking at section508 and en only, but there's the line that checks that the standard tag doesn't match more than 1 which would cover this case.